client/styles/simpleChatStyleOverwrite.css

```diff
@@ -0,0 +1,49 @@
+.box-body {
```

Comment:
If we're only using these styles in the chat, could move them to SpaceFrame/index.css?